day_3/exercises/if-statements.js

```diff
@@ -64,3 +74,15 @@ When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pi
 
 var cupsOfFlour = 1;
 var hasSauce = true;
+
+if (cupsOfFlour == 1 && hasSauce == true) {
```

Comment:
One thing I want you to get in the habit of is using stricly equal `===` over `==` - it's a better safeguard against any wonkiness that JS might interpret the wrong way. Please refactor these when you get a chance!